tests/function_call.js

```diff
@@ -24,3 +26,28 @@ describe('function call can throw',
     const result = GLib.filenameFromUri('http://google.com')
     console.log('Result:', result)
   }))
+
+describe('transfer-full function argument', () => {
+  it("doesn't invalidate js wrapper -- boxed", () => {
```

Comment:
Single quotes

Reply:
I used double quotes so that I don't have to escape the single quote in "doesn't". Should I escape it, or just use "does not"?

Reply:
Ah, ok. Yes I'd rather use single quotes everywhere.

Reply:
Fixed by using 'does not'.